src/ray/raylet/worker_pool.h

```diff
@@ -290,13 +320,13 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// the worker back onto the pool once the worker has completed its work.
   ///
   /// \param task_spec The returned worker must be able to execute this task.
-  /// \param allocated_instances_serialized_json The allocated resouce instances
-  /// json string.
-  /// \return An idle worker with the requested task spec. Returns nullptr if no
-  /// such worker exists.
-  std::shared_ptr<WorkerInterface> PopWorker(
-      const TaskSpecification &task_spec,
-      const std::string &allocated_instances_serialized_json = "{}");
+  /// \param callback The callback function that executed when gets the result of
+  /// worker popping.
+  /// \param allocated_instances_serialized_json The allocated resouce
+  /// instances json string. \return An idle worker with the requested task spec. Returns
```

Comment:
`\return Void. `
Also this comment has diverged from the interface one. I'd suggest just putting "See interface.` here.

Reply:
Good point.